### ISSUE TITLE
Replace backward slash with forward slash while creating `CheckerFrameworkError` object in `checker-framework.groovy`

### ISSUE THREAD

**Issue by Vyom-Yadav** [checkstyle/checkstyle#12524]
Currently, suppressions created in windows use backward slash because that is the path separator for windows.

Current suppression generated by windows
```xml
<checkerFrameworkError unstable="false">                                                                                                         
  <fileName>src\main\java\com\puppycrawl\tools\checkstyle\DetailAstImpl.java</fileName>                                                          
  <specifier>method.guarantee.violated</specifier>                                                                                               
  <message>@SideEffectFree method toString calls method getColumnNo with a weaker @ReleasesNoLocks side effect guarantee</message>               
  <lineContent>return text + &quot;[&quot; + getLineNo() + &quot;x&quot; + getColumnNo() + &quot;]&quot;;</lineContent>                                                        
</checkerFrameworkError>                                                                                                                         
```
Expected (expects `/`):

```xml
<checkerFrameworkError unstable="false">                                                                                                         
  <fileName>src/main/java/com/puppycrawl/tools/checkstyle/DetailAstImpl.java</fileName>                                                          
  <specifier>method.guarantee.violated</specifier>                                                                                               
  <message>@SideEffectFree method toString calls method getColumnNo with a weaker @ReleasesNoLocks side effect guarantee</message>               
  <lineContent>return text + &quot;[&quot; + getLineNo() + &quot;x&quot; + getColumnNo() + &quot;]&quot;;</lineContent>                                                        
</checkerFrameworkError>                                                                                                                         
```

All suppressions are expected to have `/` rather than `[\\/]` to keep the logic and the suppression file more uncomplicated.

**Comment by rnveach** [comment]
Fix was merged
